daemon/options/options.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2016-2017 Authors of Cilium
```

Comment:
There are some files where you have `Copyright (c) ...` and this one have `Copyright 2016-...`